### PR TITLE
Makes a software definition that just holds packages for the integrations

### PR DESCRIPTION
**Pull request by gmmeyer**

### CODE REVIEW

File: config/software/datadog-agent-integration-deps.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integration-deps'
+
+dependency 'datadog-agent'
+
+# These are the deps for the agent
```
Comment (degemer) [inline config/software/datadog-agent-integration-deps.rb]: I find this comment confusing: what do you mean by that?

File: config/projects/datadog-agent.rb [hunk 2]
```diff
@@ -193,26 +185,6 @@
 dependency 'psutil'
 dependency 'requests'
 
-unless ENV["USE_INTEGRATION_SDK"] == 'true'
```
Comment (degemer) [inline config/projects/datadog-agent.rb]: Can you keep this flag for now? (i.e. `dependency 'datadog-agent-integration-deps'`)

File: config/software/datadog-agent-integration-deps.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+dependency 'pymysql'
+dependency 'pysnmp'
+dependency 'python-memcached'
+dependency 'python-redis'
+dependency 'python-rrdtool'
+dependency 'pyvmomi'
+dependency 'scandir'
+
+dependency 'psycopg2'
+dependency 'zlib'
```
Comment (degemer) [inline config/software/datadog-agent-integration-deps.rb]: Both were not shipped with Windows previously, is this a desired change?

File: config/software/datadog-agent-integration-deps.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+if not windows?
+  dependency 'kafka-python'
+  dependency 'python-gearman'
+  dependency 'snakebite'
+else
```
Comment (truthbk) [inline config/software/datadog-agent-integration-deps.rb]: `end`?

Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: oh boy yup that's 💯

File: config/software/datadog-agent-integration-deps.rb (hunk 3 above)
Comment (degemer) [inline config/software/datadog-agent-integration-deps.rb]: Side note: zlib is a core dependency. There's a `dependency 'zlib'` in `openssl.rb`, so I think we also ship it on Windows.

File: config/projects/datadog-agent.rb (hunk 2 above)
Comment (gmmeyer) [inline config/projects/datadog-agent.rb]: sure can

File: config/software/datadog-agent-integration-deps.rb (hunk 3 above)
Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: ooo I'll put it back then!

File: config/software/datadog-agent-integration-deps.rb (hunk 1 above)
Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: Oh I meant for the integrations included with the agent

Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: I see that this is unclear, however!

File: config/projects/datadog-agent.rb (hunk 2 above)
Comment (gmmeyer) [inline config/projects/datadog-agent.rb]: I can just put it all back until we turn it all on.

File: config/software/datadog-agent-integration-deps.rb (hunk 3 above)
Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: It's not really an integration dep anyway, is it?

Comment (gmmeyer) [inline config/software/datadog-agent-integration-deps.rb]: psycopg2 is, yea, we bumped the version to a version that has a package built for windows

**Comment by gmmeyer** [comment]
I think renaming it makes sense. I'll go with `integration-dependencies` or maybe just `integration-deps`. Do you have a preference?

File: config/software/datadog-agent-integration-deps.rb (hunk 3 above)
Comment (degemer) [inline config/software/datadog-agent-integration-deps.rb]: ¯\_(ツ)_/¯

File: config/software/integration-deps.rb [hunk 5]
```diff
@@ -0,0 +1,33 @@
+require './lib/ostools.rb'
+
+name 'integration-deps'
+
+dependency 'datadog-agent'
```
Comment (gmmeyer) [inline config/software/integration-deps.rb]: You're right, it can be taken out.

**Comment by gmmeyer** [comment]
I think you're right that we should attempt to minimize these kinds of sync issues, I have moved it to this model.

**Pull request merged**